openshift/scripts/common/envars

```diff
@@ -3,8 +3,10 @@ GIT_BRANCH="${GIT_BRANCH:-$(git describe --all --contains HEAD)}"
 APP_NAME="${APP_NAME:-wps}"
 PROJ_TOOLS="${PROJ_TOOLS:-e1e498-tools}"
 PROJ_DEV="${PROJ_DEV:-e1e498-dev}"
-PROJ_PROD="${PROJ_PROD:-e1e498-prod}"
-TAG_PROD="${TAG_PROD:-prod}"
+PROJ_TEST="${PROJ_TEST:-e1e498-test}"
+PROJ_PROD="${PROJ_PROD:-e1e498-test}"
```

Comment:
Make all `PROJ_PROD` references point to `test` to minimize potential issues with updating something in prod accidentally.